packages/core/src/Services/socket-general.js

```diff
@@ -158,14 +158,18 @@ const BinarySocketGeneral = (() => {
                 if (!['reset_password', 'new_account_virtual'].includes(msg_type)) {
                     if (window.TrackJS) window.TrackJS.track('Custom InvalidToken error');
                 }
+                // eslint-disable-next-line no-case-declarations
+                const current_path = window.location.pathname;
```

Comment:
Why don't you use this function `getActivePlatform` from `platform.js` instead?

Reply:
Yep. Indeed needed this :+1: